ytools/y2tool/rubocop-1.24.1_yast_style.yml

```diff
@@ -0,0 +1,213 @@
+AllCops:
+  Exclude:
+    # avoid confusion of rpm spec and gem spec
+    - "**/*.spec"
+  TargetRubyVersion: 2.5 # 2.5 is the SLE15 version, so ensure that we use proper version
+  # By default enable all new cops
+  NewCops: enable
```

Comment:
newly added. Basically default makes updating rubocop more iterative, but it does not fit to way how we update rubocop version.